pyccel/ast/builtins.py

```diff
@@ -448,6 +456,13 @@ def args(self):
         """
         return self._args
 
+    @property
+    def allows_negative_indexes(self):
+        """ Indicates whether negative values can be
+        used to index this Variable
+        """
+        return False
```

Comment:
The docstring does not fully reflect the actual meaning of this property. `allows_negative_indexes` performs a runtime check of an index which cannot be evaluated at compile time. We may need to describe this properly throughout Pyccel...

Reply:
Fixed 8178ff88

Reply:
Thanks